### ISSUE TITLE
Request to Training Code for the Metric Learning Model

### ISSUE THREAD

**Issue by jiaor17** [Minoru938/CSPML#1]
Thank you for your excellent work!  Could you please provide the code for the architecture construction and training procedure of the metric learning model? As so far only the pre-trained parameters have been provided.
